src/appState/AppState.ts

```diff
@@ -38,8 +38,11 @@ export type ViewSplit = typeof viewSplits[number]
 
 export interface Settings {
   autoUpdateApp: boolean;
+  latexDelimiters?: 'dollars' | 'brackets' | 'doxygen' | 'gitlab' | 'julia' | 'kramdown' | 'beg_end' | 
```

Comment:
thanks for the PR!

I was just thinking whether it might make more sense for PanWriter to model these settings on pandoc's. Then it would be

```ts
extensions: {
   tex_math_single_backslash?: boolean;
}
```

corresponding to https://pandoc.org/MANUAL.html#extension-tex_math_single_backslash

Perhaps then it would even make more sense to add this to https://github.com/mb21/markdown-it-pandoc though...

Reply:
I actually started looking at doing another PR (for syntax highlighting) and noticed on that ticket that you suggested making the change in markdown-it-pandoc. I hadn't realized before that that was also your repo. So, yeah, I started thinking the same thing - that this latex delimiter code should also be in markdown-it-pandoc.

But... What would be the config mechanism if the code lived in markdown-it-pandoc?

Reply:
Cool!

> What would be the config mechanism if the code lived in markdown-it-pandoc?

I suppose we'd need to read out the `extensions.tex_math_single_backslash` from the settings files and pass it to `markdownItPandoc` as a config option [here](https://github.com/mb21/panwriter/blob/5a102eb8f93183f2ca67c1b10bb85ab2288c81dd/src/renderPreview/convertMd.ts#L6)